README.md

```diff
@@ -1,16 +1,16 @@
-# Data Science for the Common Good 2024
+
 # iNaturalist GeoModel Annotator Project
 
-Code repository for 2024 Data Science for the Common Good project with iNaturalist. 
+Run a web application for experts to annotate species range maps based on predictions from the [SINR/iNaturalist Geomodel](https://github.com/elijahcole/sinr).
+This is the code base for the iNatator project that was apart of the Data Science for the Common Good program run by the Center for Data Science at UMass Amherst. 
 
 Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov, Ozel Yilmazel 
 
-## Spatial Implicit Neural Representations for Global-Scale Species Mapping - ICML 2023
+## 🌍 Overview 
+We want to improve the predictions of the GeoModel by obtaining species range data from experts and incorporating that into the model. To this end, we ask experts that understand species ranges to select species and share their species expertise by correcting their ranges on a map. You can use this code to serve predictions from SINR models onto the map, save your annotations and interact with the annotation database. 
```

Comment:
This is kinda tricky, what we want is not to improve the GeoModel, but to provide a tool that allows experts to give feedback to the current Geomodel prodricitons, so that they can be later incorporated to improve the predictions. 

I would write something more like, for example:
"The iNaturalist team is interested in improving the predictions outputted by the Geomodel by asking experts that understand species ranges to share their knowledge and incorporate it into the Geomodel. To this end, we created a tool that allows obtaining expert feedback through annotations of species ranges in an interactive map. 
You can use this code to, for a species of your choosing, show current Geomodel predictions from SINR models onto the map, annotate the species range map considering presence and absence data, save the annotation to the database, and retrieve previous annotations from the database." Or something on those lines